workitems/l2-transaction-fees/l2-transaction-fees-v1.0-psd01.md

```diff
@@ -119,71 +121,139 @@ The L2 WG is an open-source initiative with a scope to
 
 **Layer 2:**
 
-A secondary framework or protocol that is built on top of an existing Layer 1 system in such a way that it inherits the security properties of the Layer 1 system while allowing for a higher transaction throughput that the Layer 1 system.
+A secondary framework or protocol that is built on top of an existing Layer 1 system in such a way that it inherits the security properties of the Layer 1 system while allowing for a higher transaction throughput than the Layer 1 system.
+
+**Layer 2 Operator:**
+
+An entity responsible for the development and/or operation of a Layer 2 network or single Layer 2 network node, except for entities that only provide access to a Layer 2.
+
+**Layer 2 Block:**
 
-**Maximal Extractable Value**
+A Layer 2 data object stored on a Layer 1 and typically comprised of a set of Layer 2 Transactions or their compressed representations and/or a cryptographic representation of the Layer 2 State after applying the set of Layer 2 Transactions to the Layer 2 state. This Layer 2 data object is also cryptographically linked to the previous Layer 2 Block where the cryptographic link is derived using only the previous Layer 2 Block's data.
+
+**Layer 2 Operating Condition:** 
+
+A combination of the current volume of L2 transactions waiting to be processed on an L2 and all the transactions currently being processed on an L2.
+
+**Maximal Extractable Value:**
 
 Refers to the maximum value that can be extracted from block production in excess of the standard block reward and gas fees by including, excluding, and changing the order of transactions in a block.
 
-**Priority Fee**
+**Meta Transaction:**
 
-To be paid by the transaction originator to a Layer 2 sequencer to obtain a desired slot for its transaction in a new block.
+A transaction where the Transaction Sender is not the Transaction Originator, and the transaction fee for the Transaction Originator is different if the same transaction was a direct transaction.
 
-Note that the exact position of a transaction in a new block may be determined by factors such as time-stamp, or minimization of Maximal Extractable Value (MEV) of a block in addition to a Priority Fee.
+**Priority Fee:**
+
+To be paid by the Transaction Originator or Transaction Sender to a Layer 2 sequencer to obtain a desired slot for its transaction in a new block.
 
-**Sequencer**
+Note that the exact position of a transaction in a new block may be determined by factors such as time-stamp, or minimization of Maximal Extractable Value (MEV) of a block in addition to a Priority Fee.
 
-Collects transactions, publishes them in a batch to the Layer 1 on which the Layer 2 operates, receives transaction fees from the published transactions, pays Layer 2 fees to other Layer 2 protocol participants, and, if required, participates in a consensus algorithm with other sequencers to determine transaction  ordering in a block.
+**Sequencer:**
```

Comment:
@DZGoldman reworded:

**Sequencer:**

Collects L2 transactions, publishes them in a batch to the Layer 1 on which the Layer 2 operates, and, if required, participates in a consensus algorithm with other sequencers to determine transaction ordering in a Layer 2 block. It may collect transaction fees from the published transactions, pay Layer 2 fees to other Layer 2 protocol participants, or distribute transaction fees to 3rd parties as designated.